### ISSUE TITLE
Default install gives error - orientdb-community-2.2.12

### ISSUE THREAD

**Issue by dashawn888** [orientechnologies/orientdb#6843]
## OrientDB Version, operating system, or hardware.
- orientdb-community-2.2.12
## Operating System
- Linux
## Expected behavior and actual behavior

Default install produces an error when attempting to run ./bin/[d]server.sh
## Steps to reproduce the problem

cd /tmp/
wget http://mkt.orientdb.com/CE-2212-linux
tar xfz CE-2212-linux
cd orientdb-community-2.2.12/
ps aux | grep -i java
./bin/server.sh
## Uname

Linux ubuntu 4.4.0-42-generic #62-Ubuntu SMP Fri Oct 7 23:11:45 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
## Output

```
2016-10-25 11:34:13:703 INFO  Loading configuration from: /opt/orientdb-community-2.2.12/config/orientdb-server-config.xml... [OServerConfigurationLoaderXml]
2016-10-25 11:34:14:005 INFO  OrientDB Server v2.2.12-SNAPSHOT (build 2.2.x@r2a76ee1efc4487d9dc5f43e1ddc2fba1c5a0167c; 2016-10-20 13:55:08+0000) is starting up... [OServer]
2016-10-25 11:34:14:011 INFO  Databases directory: /opt/orientdb-community-2.2.12/databases [OServer]Error during initialization of engine 'plocal', engine will be removed
java.lang.reflect.InaccessibleObjectException: Unable to make member of class com.sun.management.internal.OperatingSystemImpl accessible:  module jdk.management does not export com.sun.management.internal to unnamed module @37bb845
        at sun.reflect.Reflection.throwInaccessibleObjectException(java.base@9-internal/Reflection.java:420)
        at java.lang.reflect.AccessibleObject.checkCanSetAccessible(java.base@9-internal/AccessibleObject.java:174)
        at java.lang.reflect.Method.checkCanSetAccessible(java.base@9-internal/Method.java:189)
        at java.lang.reflect.Method.setAccessible(java.base@9-internal/Method.java:183)
        at com.orientechnologies.common.util.OMemory.getPhysicalMemorySize(OMemory.java:65)
        at com.orientechnologies.orient.core.engine.OMemoryAndLocalPaginatedEnginesInitializer.configureDefaultDiskCacheSize(OMemoryAndLocalPaginatedEnginesInitializer.java:80)
        at com.orientechnologies.orient.core.engine.OMemoryAndLocalPaginatedEnginesInitializer.configureDefaults(OMemoryAndLocalPaginatedEnginesInitializer.java:63)
        at com.orientechnologies.orient.core.engine.OMemoryAndLocalPaginatedEnginesInitializer.initialize(OMemoryAndLocalPaginatedEnginesInitializer.java:52)
        at com.orientechnologies.orient.core.engine.local.OEngineLocalPaginated.startup(OEngineLocalPaginated.java:56)
        at com.orientechnologies.orient.core.Orient.startEngine(Orient.java:858)
        at com.orientechnologies.orient.core.Orient.loadStorage(Orient.java:466)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.<init>(ODatabaseDocumentTx.java:171)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.<init>(ODatabaseDocumentTx.java:152)
        at com.orientechnologies.orient.server.OSystemDatabase.init(OSystemDatabase.java:151)
        at com.orientechnologies.orient.server.OSystemDatabase.<init>(OSystemDatabase.java:44)
        at com.orientechnologies.orient.server.OServer.initSystemDatabase(OServer.java:1280)
        at com.orientechnologies.orient.server.OServer.activate(OServer.java:350)
        at com.orientechnologies.orient.server.OServerMain$1.run(OServerMain.java:46)
Error during engine shutdown
java.lang.NullPointerException
        at com.orientechnologies.orient.core.engine.local.OEngineLocalPaginated.shutdown(OEngineLocalPaginated.java:112)
        at com.orientechnologies.orient.core.Orient.startEngine(Orient.java:864)
        at com.orientechnologies.orient.core.Orient.loadStorage(Orient.java:466)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.<init>(ODatabaseDocumentTx.java:171)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.<init>(ODatabaseDocumentTx.java:152)
        at com.orientechnologies.orient.server.OSystemDatabase.init(OSystemDatabase.java:151)
        at com.orientechnologies.orient.server.OSystemDatabase.<init>(OSystemDatabase.java:44)
        at com.orientechnologies.orient.server.OServer.initSystemDatabase(OServer.java:1280)
        at com.orientechnologies.orient.server.OServer.activate(OServer.java:350)
        at com.orientechnologies.orient.server.OServerMain$1.run(OServerMain.java:46)
com.orientechnologies.orient.core.exception.ODatabaseException: Error on opening database 'plocal:/opt/orientdb-community-2.2.12/databases/OSystem'
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.<init>(ODatabaseDocumentTx.java:191)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.<init>(ODatabaseDocumentTx.java:152)
        at com.orientechnologies.orient.server.OSystemDatabase.init(OSystemDatabase.java:151)
        at com.orientechnologies.orient.server.OSystemDatabase.<init>(OSystemDatabase.java:44)
        at com.orientechnologies.orient.server.OServer.initSystemDatabase(OServer.java:1280)
        at com.orientechnologies.orient.server.OServer.activate(OServer.java:350)
        at com.orientechnologies.orient.server.OServerMain$1.run(OServerMain.java:46)
Caused by: com.orientechnologies.orient.core.exception.OConfigurationException: Error on opening database: the engine 'plocal' was unable to start. URL was: plocal:/opt/orientdb-community-2.2.12/databases/OSystem. Registered engines was: [
memory, remote, plocal]
        DB name="plocal:/opt/orientdb-community-2.2.12/databases/OSystem"
        at com.orientechnologies.orient.core.Orient.loadStorage(Orient.java:467)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.<init>(ODatabaseDocumentTx.java:171)
        ... 6 more
```

**Comment by laa** [comment]
Hi, do you use JDK 9 ?

**Comment by dashawn888** [comment]
$ java -version
java version "9-ea"
Java(TM) SE Runtime Environment (build 9-ea+140)
Java HotSpot(TM) 64-Bit Server VM (build 9-ea+140, mixed mode)

Thanks for the help.

**Comment by dashawn888** [comment]
$ java -version
java version "9-ea"
Java(TM) SE Runtime Environment (build 9-ea+140)
Java HotSpot(TM) 64-Bit Server VM (build 9-ea+140, mixed mode)

Thanks for the help.

**Comment by tglman** [comment]
hi @dashawn888,

Java 9 hasn't been released in stable yet, we don't support it yet, and since they are changing a lot of things is not easy to keep track, in any case the first release we are testing on java 9 is the 3.0, so we don't guarantee the 2.x on java 9.

Bye

**Comment by cmassi** [comment]
I have started orientdb 2.2.12 with java 9-ea+141, with minor changes, but as @tglman said, it is still not supported, together with java 9 (early access)

**Comment by tglman** [comment]
hi @cmassi, @dashawn888 

We don't officially support it because we are not doing any work to test and fix the problems on jdk9 for version < 3.0, if you have tested and have done fixes we are really happy to accept patch on it, that we don't do it deas not mean the community shouldn't.

So really looking for to review your patches and add some support on jdk9 ;)

Bye

**Comment by lvca** [comment]
Duplicate of https://github.com/orientechnologies/orientdb/issues/6726.
